diff --git a/Cargo.lock b/Cargo.lock
index 13511c9..d9f7d7d 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -2811,7 +2811,8 @@ dependencies = [
 [[package]]
 name = "serum_dex"
 version = "0.3.1"
-source = "git+https://github.com/project-serum/serum-dex?tag=v0.3.1#7d1d41538417aa8721aabea9503bf9d99eab7cc4"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "dafc59d6c9502642898caafe8879ff7383c811cb0d6ca3a9e9b96feba9955465"
 dependencies = [
  "arrayref",
  "bincode",
diff --git a/spl/Cargo.toml b/spl/Cargo.toml
index e803e50..594a81c 100644
--- a/spl/Cargo.toml
+++ b/spl/Cargo.toml
@@ -12,6 +12,6 @@ devnet = []
 [dependencies]
 anchor-lang = { path = "../lang", version = "0.11.1", features = ["derive"] }
 lazy_static = "1.4.0"
-serum_dex = { git = "https://github.com/project-serum/serum-dex", tag = "v0.3.1", version = "0.3.1", features = ["no-entrypoint"] }
+serum_dex = { version = "0.3.1", features = ["no-entrypoint"] }
 solana-program = "1.7.4"
 spl-token = { version = "3.1.1", features = ["no-entrypoint"] }
